dls-controls/malcolmjs
# Feature/info pane #124

## Description

Added info pane component to display attribute information

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000/PANDA
- click on the 'i' icon next to any attribute (n.b. there is currently no info pane for methods)
- view the info pane!

## Agile board tracking

connect to #124 
closes #124
